app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -28,6 +28,7 @@ import cs from "./coverage_viz_bottom_sidebar.scss";
 
 const READ_FILL_COLOR = "#A9BDFC";
 const CONTIG_FILL_COLOR = "#3768FA";
+const REF_ACC_COLOR = "#EAEAEA";
```

Comment:
it's always nice to note with a comment how you chose a particular shade if it's not one of the standard shades already